docs/userguide.md

```diff
@@ -5,11 +5,20 @@ is a walkthrough tutorial for how to deploy the federation v2 control plane.
 
 ## Prerequisites
 
+The federation v2 deployment requires kubernetes version >= 1.11. The following
+is a detailed list of binaries required.
+
 ### Binaries
```

Comment:
This is same as development guide, how about keep this and let development guide link to this?

Reply:
Ideally, I'd like to have a separate `prerequisites.md` markdown file that can be imported and rendered in any number of files that you want e.g. `development.md` and `userguide.md`. But that feature does not exist AFAIK.